packages/lesmis/src/components/DashboardReportModal.js

```diff
@@ -169,7 +171,7 @@ export const DashboardReportModal = () => {
     >
       <DialogHeader
         handleClose={handleClose}
-        title={config?.dashboardName ? config?.dashboardName : 'Loading...'}
+        title={config?.dashboardName ? modalTitle : 'Loading...'}
```

Comment:
I reckon the null check and Loading switch could happen up when the const is created